cmd/webhook/main.go

```diff
@@ -81,6 +82,12 @@ func main() {
 	if namespace = os.Getenv("NAMESPACE"); namespace == "" {
 		namespace = "kube-system"
 	}
+	if *enableHealthCheck == true {
+		if *insecure == false {
```

Comment:
i think we should not implicitly set insecure to true, but rather fail and expect the user to set it WDYT ?

Reply:
I am okay with this. Will make the change.